solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -194,7 +194,7 @@ public void transform(SolrDocument doc, int docid, float score)
           if (!resultsReranked) {
             // If results have not been reranked, the score passed in is the original query's
             // score, which some features can use instead of recalculating it
-            r.setDocInfoParam(OriginalScoreFeature.ORIGINAL_DOC_SCORE, new Float(score));
+            r.docInfo.setOriginalDocScore(new Float(score));
```

Comment:
Shouldnt this be using the get() you added?  Should probably hide/privatize the member variable so people cant do this in the future